test/spmd/tensor/test_pointwise_ops.py

```diff
@@ -90,6 +90,35 @@ def test_dropout_errors(self):
                 torch.nn.functional.dropout,
             )
 
+    @with_comms
+    def test_mul(self):
+        device_mesh = DeviceMesh(self.device_type, list(range(self.world_size)))
+        torch.manual_seed(self.rank)
+        spec = [Replicate()]
+        input_tensor = torch.randn(
+            (2, 4), device=self.device_type, requires_grad=True
+        )
+        dtensor = DTensor(
```

Comment:
nit: maybe try using the `distribute_tensor` API directly? `DTensor(tensor)` API maybe a bit hard to use as we always need to pass the `requires_grad` field.